docs/guide/11-deploying.md

```diff
@@ -262,9 +262,9 @@ WORKDIR /app
 COPY --from=build /build/main ./
 
 ## copy runtime assets which may or may not exist
-COPY --from=build /build/Rocket.tom[l] ./static
-COPY --from=build /build/stati[c] ./static
-COPY --from=build /build/template[s] ./templates
+COPY --from=build /build/Rocket.tom[l] ./
+COPY --from=build /build/stati[c] ./static/
+COPY --from=build /build/template[s] ./templates/
```

Comment:
Are these changes functional?

Reply:
Yes and no. They aren't strictly needed and generally if the fix on the previous line is applied, it will work the same, but if the next two lines are applied and not the first, the error message can change, maybe depending on the `coreutils` version one uses.

I don't feel strongly about this, I thought it was a touch clearer. The first line is the important fix.